<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formSuratKeluar</class>
 <widget class="QWidget" name="formSuratKeluar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>521</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="noSuratKeluarLabel">
      <property name="text">
       <string>No Surat Keluar</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="noSuratKeluarLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="noAgendaLabel">
      <property name="text">
       <string>No Agenda</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="noAgendaLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="tglSuratLabel">
      <property name="text">
       <string>Tgl Surat</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDateEdit" name="tglSuratDateEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="asalSuratLabel">
      <property name="text">
       <string>Asal Surat</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="asalSuratLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="tujuanLabel">
      <property name="text">
       <string>Tujuan</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="tujuanLineEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="perihalLabel">
      <property name="text">
       <string>Perihal</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="perihalLineEdit"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="sifatSuratLabel">
      <property name="text">
       <string>Sifat Surat</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="sifatSuratLineEdit"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="hubSuratLabel">
      <property name="text">
       <string>Hub Surat</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="hubSuratLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>340</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Simpan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>340</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Ubah</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>340</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>340</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Cari</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableSrtKlr">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>380</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
